components/learn/modules/ROOT/pages/on-dapps.adoc

```diff
@@ -1,8 +1,160 @@
+
+```js
+test: /\.sol$/,
+use: [
+  { loader: 'json-loader' },
+  {
+    loader: '@openzeppelin/solidity-loader',
+    options: {
+      network: 'development',
+      disabled: false
+    },
+  },
+],
+```
+
+NOTE: If you are using `create-react-app`, you may need to either eject the application to modify the webpack configuration, or use https://github.com/timarney/react-app-rewired[`react-app-rewired`].
+
+Then, just load the contract Solidity file from your client code, which will go through the hot-loader. Whenever you modify your `Box` contract source, the loader will automatically upgrade the instance to the latest code for you.
+
+```js
+// Load web3 context using @openzeppelin/network
+const { networkId, lib: web3 } = useWeb3Network('http://127.0.0.1:8545');
+
+// Load contract artifact
+const Box = require('../../contracts/Box.sol');
+
+// Load deployed instance
+const [box, setBox] = useState(undefined);
+if (!box && Box && Box.networks && networkId) {
+  const deployedNetwork = Box.networks[networkId.toString()];
+  if (deployedNetwork) {
+    setBox(new web3.eth.Contract(Box.abi, deployedNetwork.address));
+  }
+}
+```
+
+You can follow a step-by-step tutorial on enabling hot loader in a React web application at https://forum.openzeppelin.com/t/building-an-openzeppelin-dapp-with-solidity-hot-loader/1843/1[the forum].
+
+[[starter-kit]]
+== Kickstarting your Dapp
+
+As you have seen so far, setting up a dapp may involve a certain deal of boilerplate and configuration, such as creating an OpenZeppelin CLI project, initializing a new webpack client app, setting up network configuration, loading your contracts, etc.
+
+To kickstart this process, we have built the xref:starter-kits[OpenZeppelin Starter Kits]. Starter Kits are preconfigured dapps with several OpenZeppelin libraries, Infura connections, and https://github.com/ConsenSys/rimble-ui[Rimble UI components], ready to start developing right away.
+
+You can start a new project from a starter kit using the `oz unpack` CLI command:
+
+```console
+$ npx oz unpack starter
+✓ Kit downloaded and unpacked
+The kit is ready to use. 
+
+Quick Start
+Run your local blockchain:
+> ganache-cli --deterministic
+Initialize the OpenZeppelin project:
+> openzeppelin init app
+Go to the client directory:
+> cd client
+Run the React app:
+> npm run start
+Continue in your browser!
+More at https://github.com/OpenZeppelin/starter-kit/tree/stable
+```
+
+This will unpack a new preconfigured React dapp, with a network connection set up to both metamask and the local node.
+
+image::StarterKit.png[OpenZeppelin Starter Kit,500]
+
+If you want to learn more about how to use the starter kit, `unpack` the `tutorial` instead of the `starter` kit, which will guide you through the process of deploying and interacting with contracts from a dapp. You can also check out the xref:starter-kits:list[list of all starter kits] available to unpack.
```

Comment:
```suggestion
If you want to learn more about using Starter Kits, `unpack` the xref:starter-kits::tutorial.adoc[`tutorial`] kit instead of `starter`: it will guide you through the process of deploying and interacting with contracts from a dapp. You can also check out the xref:starter-kits::list.adoc[list of all starter kits] available to unpack.
```